### PR TITLE
Admin default topics

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - will you take a look at this, see what you think.

**Comment by levity** [comment]
... s/def/is_default/ ? :)

File: api/controllers/CommunityController.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const welcomeMessage = 'Thank you for joining us here at Hylo. ' +
 
 module.exports = {
   find: function (req, res) {
-    Community
+    return Community
```
Comment (levity) [inline api/controllers/CommunityController.js]: why did you modify Community.find? as far as i can tell it's no longer used by the front-end

**Comment by levity** [comment]
:cool:

**Pull request merged**